src/os_auth/auth.c

```diff
@@ -0,0 +1,326 @@
+/* Auth Common
+ * Copyright (C) 2015-2020, Wazuh Inc.
+ * Mar 22, 2018.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <shared.h>
+#include "auth.h"
+#include "os_err.h"
+
+#ifdef UNIT_TESTING
+#define static
+#endif
+
+/*authd working as worker on cluster*/
+bool worker_node;
```

Comment:
This flag is declared in auth.c because modifies the behaviour of the complete module, beside it's only used on main-server.
In future: This flag should be inside a cfg struct of the module.